i3lock.c

```diff
@@ -1038,6 +1038,8 @@ int main(int argc, char *argv[]) {
         err(EXIT_FAILURE, "getpwuid() failed");
     if ((username = pw->pw_name) == NULL)
         errx(EXIT_FAILURE, "pw->pw_name is NULL.");
+    if getenv("WAYLAND_DISPLAY")
```

Comment:
I don’t think this compiles? I thought parens are required in C.

Please make this read:

```C
if (getenv("WAYLAND_DISPLAY") != NULL) {
  errx(EXIT_FAILURE, "i3lock is a program for X11 and does not work on Wayland. Try https://github.com/swaywm/swaylock instead");
}
```

Reply:
You're right. I was just ammending, long time I don't do C.